app/views/users/new.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<% if @user.errors.any?  %>
+    <h3 style="color:red">>we couldn't submit your form for these reasons</h3>
+    <% @user.errors.full_messages.each do |msg| %>
+    <li style="color:red"><%= msg %></li>
+    <% end %>
+<% end %>
+<%= form_for @user do |f| %>
+  <%= f.label :username, 'Your username here :'%>
+  <%= f.text_field :username, placeholder: "name1234" %><br><br>
+  <%= f.label :username, 'Your Email here :'%>
+ <%= f.text_field :email, placeholder: "example@email.com" %><br><br>
+ <%= f.label :username, 'Your  Password here :'%>
+  <%= f.password_field :password, placeholder: "password" %><br><br>
+  <%= f.submit "Create" %>
+<% end %>
```

Comment:
You are missing the two previous commented out forms.
You ought to leave the two previous forms you created as comments.

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) and this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) concerning the HTML form

And this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png)